library/src/scripts/textEditor/TextEditor.tsx

```diff
@@ -25,11 +28,12 @@ export default function TextEditor(props: ITextEditorProps) {
             <button onClick={toggleTheme} className={classes.themeToggleIcon} disabled={!isEditorReady}>
                 <ToggleIcon />
             </button>
-            <Editor
+            <ControlledEditor
                 theme={intialTheme}
                 language={language}
                 editorDidMount={handleEditorDidMount}
```

Comment:
do we still need editorDidMount prop?

Reply:
yes, if we remove this, it affect theme toggle button event.